flasksrc/user.py

```diff
@@ -1,33 +1,42 @@
 import datetime
 import argon2
 from marshmallow import Schema, fields, ValidationError
-
 from flask import request, current_app
 from flask_restful import Resource, abort
 from flasksrc.db import get_db
 from flask_jwt_extended import create_access_token
+from flask_apispec import marshal_with
 
 
 hasher = argon2.PasswordHasher()
 
 
-class ToyNetUserReq(Schema):
-    """ /api/user - POST
-
-    Parameters:
-     - password (str)
-     - username (str)
-     - first_name (str, optional)
-    """
+# Schema definitions
+class ToyNetUserPostReq(Schema):
     password = fields.Str(required=True)
     username = fields.Str(required=True)
     first_name = fields.Str(required=False)
 
 
+class ToyNetUserPostResp(Schema):
+    username = fields.Str(required=True)
+
+
+class ToyNetUserLoginPostReq(Schema):
+    username = fields.Str(required=True)
+    password = fields.Str(required=True)
+
+
+class ToyNetUserLoginPostResp(Schema):
+    verified = fields.Bool(required=True)
```

Comment:
What is the purpose of verified here?

Reply:
I think this is intended to give an additional indicator of success to the frontend on the user status? @takakonishimura ?